source/Refitting/src/ExtrToSIT.cc

```diff
@@ -217,7 +221,15 @@ void ExtrToSIT::init() {
   SITHitsNonFitted = 0 ;
   TotalSITHits = 0 ;
 
-  _bField = Global::GEAR->getBField().at( gear::Vector3D(0., 0., 0.) ).z();    //The B field in z direction
+  DD4hep::Geometry::LCDD& lcdd = DD4hep::Geometry::LCDD::getInstance();
```

Comment:
... yes, in the init() for now. Eventually this will need to be replaced with a consistent treatment of the LCDD initialization in Marlin, e.g. by replacing ```init()``` with ```init( DD4hep::LCDD& lcdd)``` and then no more calls to LCDD::instance() ...